README.rst

```diff
@@ -3,4 +3,78 @@ Aexpect: Control your interactive applications
 
 This module provides services similar to the `pexpect` python library,
```

Comment:
Could we phrase this as "this project"? After all, this is documentation for the entire project and aexpect is not a single module for quite a while now if since the very beginning (having more than just `client` module).

Reply:
ack